scipy/signal/_spectral_py.py

```diff
@@ -13,41 +13,50 @@
            'spectrogram', 'stft', 'istft', 'check_COLA', 'check_NOLA']
 
 
-def lombscargle(x,
-                y,
-                freqs,
-                precenter=False,
-                normalize=False):
+def lombscargle(
+    t: npt.NDArray,
+    y: npt.NDArray,
```

Comment:
Could we leave these as x and y? otherwise, this breaks backwards compatibility

Reply:
I was trying to fix this nomenclature bug. But, yeah, I can revert `t` back to `x`.

Reply:
The other option is to use the following to rename
https://github.com/scipy/scipy/blob/a58f4b61557b9ca649fd63d05338eee368efce41/scipy/_lib/_util.py#L762-L814

Reply:
Reverted back to `x`.